File: python/shuffle.py
#!/usr/bin/python

import sys

if len( sys.argv ) < 3:
  print"mooop"
  sys.exit(-1)

try:
  fin1 = open( sys.argv[1], 'rt' )
  fin2 = open( sys.argv[2], 'rt' )
except:
  exit(-1)
  

for line1 in fin1:

  for line2 in fin2:
  
    sys.stdout.write( line1.strip("\n") )
    sys.stdout.write( line2 )

  fin2.seek(0)

fin1.close()
fin2.close()

File: lua/shuffle.lua
#!/usr/bin/lua

if arg[1] == nil then
  return
end

fin1 = io.open( arg[1], "r" )
if fin1 == nil then
  return
end

if arg[2] == nil then
  return
end

fin2 = io.open( arg[2], "r" )
if fin2 == nil then
  return
end



for line1 in fin1:lines() do
	for line2 in fin2:lines() do
		io.write( line1 .. line2 .. "\n" )
	end
	fin2:seek( "set", 0 );
end
